demisto_sdk/commands/common/tools.py

```diff
@@ -182,17 +182,24 @@ def get_child_files(directory):
     return child_files
 
 
-def get_last_release_version():
+def get_last_remote_release_version():
     """
-    Get latest release tag (xx.xx.xx)
+    Get latest release tag from remote github page
 
     :return: tag
     """
-    tags = run_command('git tag').split('\n')
-    tags = [tag for tag in tags if re.match(r'\d+\.\d+\.\d+', tag) is not None]
-    tags.sort(key=LooseVersion, reverse=True)
-
-    return tags[0]
+    try:
+        releases_request = requests.get(SDK_API_GIHUB_RELEASES, verify=False)
+        releases_request.raise_for_status()
+        releases = requests.get(SDK_API_GIHUB_RELEASES, verify=False).json()
+        if isinstance(releases, list) and isinstance(releases[0], dict):
+            latest_release = releases[0].get('tag_name')
+            if isinstance(latest_release, str):
+                # remove v prefix
+                return latest_release[1:]
+    except Exception:
+        print_warning('Could not get latest demisto-sdk version.')
```

Comment:
I think you should print in the second line the error that you get in the exception, and especially check it when you are disconnected from the internet. make sure the error is readable and add extra handling for this case if it is not.

Reply:
![image](https://user-images.githubusercontent.com/20818773/76166296-33182400-6166-11ea-9a2d-1c8a42c4debc.png)